src/LuaInterface.cpp

```diff
@@ -45,7 +45,7 @@ int LuaInterface::onPanic(lua_State* L)
 {
     QString error = "Lua Panic, No error information";
     if (lua_isstring(L, -1)) {
-        error = QString::fromUtf8(lua_tostring(L, -1));
```

Comment:
:warning: Please check and confirm that the new version still compiles if `QT_NO_CAST_FROM_ASCII` is `#define`d, whilst the implicit construction of a `QString` from a `const char*` will also use the `QString::fromUtf8(...)` method it will automatically cause a compile failure then as that becomes a private constructor - this is so that all `QString`s are *marked* as being EITHER: for the UI (and thus must be translated by being wrapped in `QObject::tr(...)` or `QApplication::translate(...)`)  OR: for internal use (and thus must be wrapped in `QLatin1String(...)` or `QStringLiteral(...)`).